Fix one or more forms, affected by some update in Flask during last week or so

On a clean installation , python=3.6, there is at least a test failing:
tests/server/blueprints/variants/test_variants_utils.py::test_modal_prompt_filter_name 

with this message:
```
cls = <class 'wtforms.fields.core.IntegerField'>, validators = [<wtforms.validators.Optional object at 0x109cfaa20>, <class 'wtforms.fields.core.IntegerField'>]

    @classmethod
    def check_validators(cls, validators):
        if validators is not None:
            for validator in validators:
                if not callable(validator):
                    raise TypeError("{} is not a valid validator because it is not "
                                    "callable".format(validator))
    
                if inspect.isclass(validator):
                    raise TypeError("{} is not a valid validator because it is a class, "
>                                   "it should be an instance".format(validator))
E                   TypeError: <class 'wtforms.fields.core.IntegerField'> is not a valid validator because it is a class, it should be an instance

../../../../miniconda3/envs/appo/lib/python3.6/site-packages/wtforms/fields/core.py:172: TypeError
```

This is likely to reflect on the web interface as well. Do not freeze Flask but fix the WTF (no, not that one, the flask one) because something in there must have become obsolete.

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -11,6 +11,8 @@ About changelog [here](https://keepachangelog.com/en/1.0.0/)
 - Adds verification comment box to SVs (previously only available for small variants)
 
 ### Fixed
+- Error caused by changes in WTForm (new release 2.3.x)
+
 ### Changed
 
 

--- a/scout/server/blueprints/variants/forms.py
+++ b/scout/server/blueprints/variants/forms.py
@@ -100,8 +100,8 @@ class FiltersForm(VariantFiltersForm):
     spidex_human = SelectMultipleField("SPIDEX", choices=SPIDEX_CHOICES)
 
     chrom = TextField("Chromosome", [validators.Optional()])
-    start = IntegerField("Start position", [validators.Optional(), IntegerField])
-    end = IntegerField("End position", [validators.Optional(), IntegerField])
+    start = IntegerField("Start position", [validators.Optional()])
+    end = IntegerField("End position", [validators.Optional()])
     local_obs = IntegerField("Local obs. (archive)")
 
     filter_variants = SubmitField(label="Filter variants")
@@ -140,8 +140,8 @@ class SvFiltersForm(VariantFiltersForm):
     swegen = IntegerField("SweGen obs")
 
     chrom = TextField("Chromosome", [validators.Optional()])
-    start = IntegerField("Start position", [validators.Optional(), IntegerField])
-    end = IntegerField("End position", [validators.Optional(), IntegerField])
+    start = IntegerField("Start position", [validators.Optional()])
+    end = IntegerField("End position", [validators.Optional()])
 
     filter_variants = SubmitField(label="Filter variants")
     clinical_filter = SubmitField(label="Clinical filter")

--- a/scout/server/blueprints/variants/forms.py
+++ b/scout/server/blueprints/variants/forms.py
@@ -43,8 +43,7 @@ class TagListField(Field):
     def _value(self):
         if self.data:
             return ", ".join(self.data)
-        else:
-            return ""
+        return ""
 
     def process_formdata(self, valuelist):
         if valuelist:

